examples/scripts/reward_modeling.py

```diff
@@ -98,7 +99,7 @@ class ScriptArguments:
 
 # Step 2: Load the dataset and pre-process it
 tokenizer = AutoTokenizer.from_pretrained(args.model_name)
-train_dataset = load_dataset(args.dataset_name, split="train")
+train_dataset = load_dataset(args.dataset_name, split="train[:50]")
```

Comment:
```suggestion
train_dataset = load_dataset(args.dataset_name, split="train")
```
I used that for testing purpose, will remove it!